### PR TITLE
Fix contributing file and set php version

### PR DESCRIPTION
**Pull request by VincentLanglet**
I followed https://github.com/symfony/flex/issues/609 for the php-version issue I had. (Every time I used composer, the symfony.lock was updated from 7.3 to 7.4)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 Thanks for you interest onto Sonata projects!
 
-Want to contribute? Please read the following documentation first:
-[project/CONTRIBUTING.md](project/CONTRIBUTING.md).
+Want to contribute? Please read the [following documentation](templates/project/CONTRIBUTING.md) first.
 
 This is the only documentation you have to refer.
```
Comment (greg0ire) [inline CONTRIBUTING.md:7]: For bonus points:
```suggestion
This is the only documentation you have to refer to.
```

Comment (VincentLanglet) [reply CONTRIBUTING.md:7]: That's not fair ! I get a mistake on something I didn't even write 😭

Comment (greg0ire) [reply CONTRIBUTING.md]: For bonus points I said! You don't have to do it :P

Comment (VincentLanglet) [reply CONTRIBUTING.md]: Done ;)

**Comment by greg0ire** [comment]
Thanks @VincentLanglet !

**Comment by OskarStark** [comment]
Thanks, but we can upgrade to 7.4

**Pull request merged**